Stripe/Resources/Localizations/es.lproj/Localizable.strings

```diff
@@ -1,170 +1,153 @@
+"Enter card details manually" = "Escribe los datos de la tarjeta manualmente.";
 
 /* No comment provided by engineer. */
-"Enter the verification code to use the payment info you stored with Stripe." = "Enter the verification code to use the payment info you stored with Stripe.";
+"Enter the verification code to use the payment info you stored with Stripe." = "Escribe el código de verificación para usar la información de pago que tiene almacenada Stripe.";
 
 /* accessibility label for text field */
-"expiration date" = "expiration date";
+"expiration date" = "fecha de vencimiento";
 
 /* Error message for checkout account api call */
-"Failed to parse account response" = "Failed to parse account response";
+"Failed to parse account response" = "Error al analizar la respuesta de la cuenta.";
 
 /* Cancel button for Remember Me SMS verification screen. */
-"Fill in your card details manually" = "Fill in your card details manually";
+"Fill in your card details manually" = "Rellena los datos de la tarjeta manualmente.";
 
 /* Message shown when the user enters an incorrect SMS verification code. */
-"Invalid Code" = "Invalid Code";
-
-/* Placeholder for Name field on address form */
-"John Appleseed" = "John Appleseed";
+"Invalid Code" = "Código no válido";
 
 /* Title for screen when data is still loading from the network. */
-"Loading…" = "Loading…";
+"Loading…" = "Cargando…";
 
 /* Caption for Name field on address form */
-"Name" = "Name";
+"Name" = "Nombre";
 
 /* Button to move to the next text entry field */
-"Next" = "Next";
+"Next" = "Siguiente";
 
 /* No comment provided by engineer. */
-"OK" = "OK";
+"OK" = "Aceptar";
 
 /* Button to paste a copied SMS code into the verification field. */
-"Paste copied code?" = "Paste copied code?";
+"Paste copied code?" = "¿Pegar el código copiado?";
 
 /* Title for Payment Method screen */
-"Payment Method" = "Payment Method";
+"Payment Method" = "Método de pago";
 
 /* Caption for Phone field on address form */
-"Phone" = "Phone";
+"Phone" = "Teléfono";
 
-/* Caption for Postal Code field on address form (non-US regions only) */
-"Postal Code" = "Postal Code";
+/* Caption for Postal Code field on address form (only shown in countries other than the United States) */
+"Postal Code" = "Código postal";
 
-/* Placeholder for City field on address form */
-"San Francisco" = "San Francisco";
+/* Caption for Province field on address form (only countries that use province, like Canada) */
+"Province" = "Provincia";
 
 /* Label for the switch to enable Remember Me */
-"Save for use in other apps" = "Save for use in other apps";
+"Save for use in other apps" = "Guardar para otras aplicaciones";
 
 /* Error message for Remember Me network error. More detailed cause of the error will be filled into the substitution. */
-"Something went wrong with Remember Me: %@" = "Something went wrong with Remember Me: %@";
+"Something went wrong with Remember Me: %@" = "La función \"Recordarme\" no ha funcionado bien: %@";
```

Comment:
nitpick: we should use typographic quotes rather than straight quotes 
here ya go (>'_')> “ ”